server/actions/serveFile.js

```diff
@@ -21,184 +21,229 @@ const AutoIndex = !process.env.DISABLE_INDEX;
 const MaximumDepth = 128;
 
 function serveMetadata(req, res) {
-  getMetadata(
-    req.packageDir,
-    req.filename,
-    req.stats,
-    MaximumDepth,
-    (error, metadata) => {
-      if (error) {
-        console.error(error);
-
-        res
-          .status(500)
-          .type("text")
-          .send(
-            `Cannot generate metadata for ${req.packageSpec}${req.filename}`
-          );
-      } else {
-        // Cache metadata for 1 year.
-        res
-          .set({
-            "Cache-Control": "public, max-age=31536000",
-            "Cache-Tag": "meta"
-          })
-          .send(metadata);
-      }
-    }
-  );
+	getMetadata(
```

Comment:
Please leave the indentation at 2 spaces so the diff isn't so large. It makes it easier for me to see where the changes are being made. Thank you!

Reply:
Yeah, sure :+1: